global/meta/models.yml

```diff
@@ -111,6 +111,12 @@ organization:
     restriction_mode: B
     default: 0
     minimum: 0
+  default_language:
```

Comment:
make it required to enforce users to give it in the initial import

Reply:
I have made orga default_language required.